packages/transaction-log/src/services/api.service.ts

```
@@ -80,3 +81,47 @@ export async function getServerHeartBeat(): Promise<{
+}
+
+export async function getBulkRunFilterCount(params: {
+  filterLimit: number
+  filterPage: number
+  filterRepresentation: string
+  filters: {}
+}): Promise<{count: number}> {
+  try {
+    return await fetchBulkRunFilterCount(params)
+  } catch (error) {
+    throw error
```

Biome

[error] The catch clause that only rethrows the original error is redundant.

These unnecessary catch clauses can be confusing. It is recommended to remove them.

(lint/complexity/noUselessCatch)